test/pylib/manager_client.py

```diff
@@ -175,3 +177,19 @@ async def server_update_config(self, server_id: str, key: str, value: object) ->
     async def get_host_id(self, server_id: str) -> str:
         """Get host id through Scylla REST API"""
         return await self.api.get_host_id(server_id)
+
+    async def wait_for_host_known(self, dst_server_id: str, expect_host_id: str, deadline: float = None) -> None:
```

Comment:
`None` is not a valid value for `float`.
You can make the type `Optional[float]`.

Although I would personally prefer not to make decisions such as timeout length inside these API definitions, forcing the caller to pass the timeout instead.